src/templates/globals/util/breadcrumbUtils.tsx

```diff
@@ -0,0 +1,65 @@
+import { BreadcrumbItem } from '@/types/index'
+
+export function deriveLastBreadcrumbFromPath(
+  breadcrumbs: BreadcrumbItem[],
+  breadcrumbTitle: string,
+  currentPath: string,
+  replaceLastItem: boolean
+): BreadcrumbItem[] {
+  const last: BreadcrumbItem = {
+    uri: currentPath,
+    title: breadcrumbTitle,
+    options: [],
+  }
+
+  if (replaceLastItem) {
+    breadcrumbs.splice(breadcrumbs.length - 1, 1, last)
+  } else {
+    breadcrumbs.push(last)
+  }
+
+  return breadcrumbs
+}
+
+export function deriveLcBreadcrumbs(
```

Comment:
What is "Lc"? I found myself wondering this a few times when looking through. I wonder if it could be added in a comment somewhere. Or maybe I'm just really blanking on something very obvious 😅

Reply:
you know, i just assumed it was 'learning center' or something else I hadn't worked with yet and breezed past that,  I'm actually not sure.

Reply:
for the record it is learning centers, here is the PR it originates from: https://github.com/department-of-veterans-affairs/vets-website/pull/14687

Reply:
Added a comment.  When we add said content types we can re-evaluate if this logic is actually necessary.